go.mod

```diff
@@ -7,17 +7,17 @@ require (
 	github.com/fatih/color v1.9.0
 	github.com/gavv/httpexpect v2.0.0+incompatible
 	github.com/gocolly/colly/v2 v2.0.1
-	github.com/gorilla/mux v1.7.4
+	github.com/gorilla/mux v1.8.0
 	github.com/gorilla/websocket v1.4.2
 	github.com/mattermost/ldap v3.0.4+incompatible // indirect
-	github.com/mattermost/mattermost-server/v5 v5.26.2
+	github.com/mattermost/mattermost-server/v5 v5.3.2-0.20201216152535-675fab8ab63a
```

Comment:
Waiting for https://github.com/mattermost/mattermost-server/pull/16503 to be merged to update this.

Reply:
@streamer45 - server PR has been merged

Reply:
Brilliant :tada: